Dockerfile

```diff
@@ -19,10 +24,54 @@ RUN yum -y install \
     freetype-devel \
     libart_lgpl-devel \
     gcc \
-    groff
+    groff \
+    wget \
+    automake \
+    libffi-devel \
+    libyaml-devel
+
+# RUN cd tmp && \
+#     wget https://www.openssl.org/source/openssl-1.0.2a.tar.gz && \
+#     tar -zxvf openssl-*.tar.gz && \
+#     cd openssl-*/ && \
+#     ./config -fpic shared && make && make install && \
+#     echo "/usr/local/ssl/lib" >> /etc/ld.so.conf && \
+#     ldconfig
+
+RUN yum -y install \
+    make \
+    automake \
+    autoconf \
+    install \
+    perl-ExtUtils-MakeMaker \
+    fakeroot
```

Comment:
nit: the package list should probably be alphabetically sorted. I'm also unclear why this group is separate from the first install. Either way, you want a `yum update` here inlined too to ensure that the build doesn't fail when only this layer gets invalidated.